CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Added `thiserror` dependency for custom error types ([#18](https://github.com/stjude-rust-labs/tes/pull/18)).
```

Comment:
Nit: this seems like a revision to me instead of an added (the error trait implementation already existed, we're just changing it to be derived).

Reply:
Probably "Changed" should be the section if you do update that.